index.js

```diff
@@ -52,6 +52,11 @@ GulpRunner.prototype.run = function(tasks, options, cb) {
       cb(null)
     }
   });
+
+  process.on('SIGINT', function() {
+    gulp.kill();
+    process.exit(1);
```

Comment:
`SIGINT` is basically fired when `Ctrl + C` is pressed in the middle of a gulp task. So I think "fail" is appropriate! As info, it doesn't actually show a failure code or stack trace, so merely provided to use the correct code!